src/main/java/org/cookieandkakao/babting/domain/meeting/controller/MeetingController.java

```diff
@@ -71,12 +86,50 @@ public ResponseEntity<SuccessBody<Void>> exitMeeting(
         return ApiResponseGenerator.success(HttpStatus.OK, "모임 탈퇴 성공");
     }
 
-    // 참여 모임 목록 조회
+    // 내가 참여한 모임 목록 조회
     @GetMapping
+    @Operation(summary = "내가 참여한 모임 목록 조회", description = "내가 참여한 모임의 목록을 가져옵니다.")
+    @ApiResponse(responseCode = "200", description = "참여 모임 목록 조회 성공")
     public ResponseEntity<SuccessBody<List<MeetingGetResponse>>> getAllMeeting(
         @LoginMemberId Long memberId
     ){
         List<MeetingGetResponse> meetings = meetingService.getAllMeetings(memberId);
         return ApiResponseGenerator.success(HttpStatus.OK, "참여 모임 목록 조회 성공", meetings);
     }
+
+    // 모임 id로 이름, 모임 시작과 끝 날짜 조회
+    @GetMapping("/{meetingId}")
+    @Operation(summary = "모임 정보 조회", description = "모임의 제목, 모임의 시작과 끝 날짜를 조회합니다.")
+    @ApiResponse(responseCode = "200", description = "모임 정보 조회 성공")
+    public ResponseEntity<SuccessBody<MeetingInfoGetResponse>> getMeetingInfo(
+        @PathVariable("meetingId") Long meetingId
+    ){
+        MeetingInfoGetResponse meetingInfo = meetingService.getMeetingInfo(meetingId);
+        return ApiResponseGenerator.success(HttpStatus.OK, "모임 정보 조회 성공", meetingInfo);
+    }
+    // 모임 주최자 확인
+    @GetMapping("/{meetingId}/is-host")
+    @Operation(summary = "모임 주최자 확인", description = "해당 meetingid의 모임 주최자인지 확인합니다.")
+    @ApiResponse(responseCode = "200", description = "주최자 여부 확인 성공")
+    public ResponseEntity<SuccessBody<MeetingHostCheckResponse>> getIsHost(
+        @LoginMemberId Long memberId,
```

Comment:
```suggestion
        @Parameter(hidden = true) @LoginMemberId Long memberId,
```
memberId는 swagger 문서에 안보이게 설정해주면 좋을 것 같아요